UnitTests/FeedStrategyTests.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  FeedStrategyTests.swift
+//  UnitTests
+//
+//  Created by Matthew Lorentz on 5/17/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import XCTest
+
+// swiftlint:disable force_unwrapping function_body_length
+
+/// Tests to make sure our `FeedStrategy`s fetch the posts we expect.
+class FeedStrategyTests: XCTestCase {
+    
+    var tmpURL = URL(string: "unset")!
+    var db = ViewDatabase()
+    let testAuthor: Identity = DatabaseFixture.exampleFeed.identities[0]
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        db.close()
+        
+        // get random location for the new db
+        self.tmpURL = URL(fileURLWithPath: NSTemporaryDirectory().appending("/viewDBtest-feedFill2"))
+        
+        do {
+            try FileManager.default.removeItem(at: self.tmpURL)
+        } catch {
+            // ignore - most likely not exists
+        }
+        
+        try FileManager.default.createDirectory(at: self.tmpURL, withIntermediateDirectories: true)
+        
+        // open DB
+        let dbPath = tmpURL.absoluteString.replacingOccurrences(of: "file://", with: "")
+        try db.open(path: dbPath, user: testAuthor, maxAge: -60 * 60 * 24 * 30 * 48)
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        db.close()
+    }
+
+    func testPatchworkFeedStrategy() throws {
```

Comment:
Wow, I like this way of testing the strategy more that the way I did it.

Reply:
Maybe when we refactor the strategies to share more code we could refactor the tests too.

Reply:
Left a comment there